_episodes/09-documentation.md

```diff
@@ -412,4 +412,45 @@ conda:
 Commit to these changes and push to your repository. You should now see your documentation strings.
 
 
-{% include links.md %}
\ No newline at end of file
+#### Automatically Generating Documentation Using Sphinx-AutoAPI
```

Comment:
The header for this section should be two `##`

Reply:
Added in latest commit.